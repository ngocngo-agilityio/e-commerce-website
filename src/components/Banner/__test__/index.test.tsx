// Libs
import { render } from '@testing-library/react';

// Components
import SaleBanner from '..';

describe('SaleBanner components', () => {
  test('should render CartTableHeader successfully', () => {
    const { container } = render(<SaleBanner />);

    expect(container).toBeInTheDocument();
  });

  test('should match snapshot for SaleBanner', () => {
    const { container } = render(<SaleBanner />);

    expect(container).toMatchSnapshot();
  });
});
