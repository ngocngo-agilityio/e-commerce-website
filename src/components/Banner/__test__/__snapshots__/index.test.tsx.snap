// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SaleBanner components should match snapshot for SaleBanner 1`] = `
<div>
  <div
    class="css-1xt0cvq"
  >
    <div
      class="chakra-container css-0"
    >
      <div
        class="css-ckyhk3"
      >
        <div
          class="css-cpk5wh"
        >
          <h2
            class="chakra-heading css-rrcadk"
          >
            stylist picks beat the heat
          </h2>
          <button
            class="chakra-button css-1u2i1wc"
            type="button"
          >
            SHOP NOW
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
